deps/libgit2.mk

```diff
@@ -40,6 +40,10 @@ ifeq ($(OS),Linux)
 LIBGIT2_OPTS += -DUSE_OPENSSL=OFF -DUSE_MBEDTLS=ON -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
 endif
 
+ifeq ($(OS),FreeBSD)
+LIBGIT2_OPTS += -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
```

Comment:
Bummers. It would be good to track that down eventually, for consistency's sake. The reasons that we switched to mbedtls on Linux also apply on FreeBSD I believe, but that can wait until someone has the bandwidth to investigate it.